CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Cast
 
+#### Changed
+
+- `--name` flag is now optional when using `account create` (default name is generated) 
```

Comment:
In fact, this should be listed below. Have a look, there's already a `####Changed```` section below. So please put it there.